mypy/fastparse.py

```diff
@@ -95,7 +95,8 @@ def with_line(f: Callable[['ASTConverter', T], U]) -> Callable[['ASTConverter',
     @wraps(f)
     def wrapper(self: 'ASTConverter', ast: T) -> U:
         node = f(self, ast)
-        node.set_line(ast.lineno)
+        # some ast nodes (e.g. Return) do not come with col_offset
```

Comment:
Heh, I figured out why (by putting a pdb trap here). It's because visit_lambda() synthesizes a Return node and sets only the lineno. Once you fix that I think the getattr() call is no longer necessary (and it shouldn't be!).